deployments/daemon-set/nginx-ingress.yaml

```diff
@@ -65,7 +65,8 @@ spec:
         securityContext:
           allowPrivilegeEscalation: false
 #          readOnlyRootFilesystem: true
-          runAsUser: 101 #nginx
+          runAsUser: null
```

Comment:
could this just be removed? instead of `null`?

Reply:
I was thinking it might be preferred to explicitly say "this is undefined".
But yes - should be safe to drop the line outright, allowing the default behavior to kick in automatically.